### DISCUSSION TITLE
Boat Physics - Adding colliders and player to boat causing big issues

### DISCUSSION THREAD

**Discussion by JackkFi5h** [crest-ocean/crest#1103] [Q&A]
Basically, I am using the large boat from the threeboats example provided with crest. My player is a character controller (no rigidbody just a capsule collider), when it is walking into a collider this happens...

https://i.gyazo.com/1939f40910afcab722910376380d1e28.mp4

The wall it collides with also has no rigidbody so I'm quite confused as to what's going on
The wall colliders are a child of the boat so I'm guessing maybe theres some function that gets all colliders possibly? Even then i dont see how the players interaction with it would cause this to happen as both the player and wall have no rigidbody

**Comment by magenta404** [comment]
This is a problem with using moving non rigid body colliders on moving rigid body colliders. It's just a matter of solving that problem.
